nav2_util/include/nav2_util/geometry_utils.hpp

```diff
@@ -62,6 +63,15 @@ inline double euclidean_distance(
   return euclidean_distance(pos1.pose, pos2.pose);
 }
 
+inline double calculate_path_length(const nav_msgs::msg::Path & path)
```

Comment:
Good thinking adding this to the utils :+1:

Reply:
thanks :)